edgexfoundry/device-rfid-llrp-go
# feat!: Consume SDK interface changes

BREAKING CHANGE: Consume SDK interface changes by adding Discover and ValidateDevice func on driver







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-rfid-llrp-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

- Run core service and device service to verify func work
- Run unittest

## New Dependency Instructions (If applicable)
